app/src/main.cpp

```
@@ -12,6 +12,7 @@ Wireless client;
 void setup()
 {
   Serial.begin(9600);
+  Serial.println("ASd")
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If println is a macro then please configure it.

(unknownMacro)